src/infra/monitoring/grafana/healthmodelpanel/package.json

```diff
@@ -14,7 +14,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@grafana/data": "^8.2.2",
-    "@grafana/toolkit": "^8.3.4",
+    "@grafana/toolkit": "^8.3.5",
```

Comment:
shouldnt this not be updated? (and the healthmodelpanel/yarn.lock)

Reply:
this is already the previous update #69, #70 and #71 contain already the next update

Reply:
`8.3.5` to `8.3.6`

Reply:
we need a way to properly test that in e2e now that grafana is in main